platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -81,135 +81,113 @@ identity access, infrastructure operations, and app lifecycle.
-
-At its most basic, a platform provides consistent experiences to application 
-developers for acquiring and using individual services such as a database system 
-or a secret store. Example use cases an enterprise can meet with a basic platform 
-include the following:
-
-1. Developers of products and services can automatically provision runtime 
-   capabilities such as compute, storage, databases and service identity and 
-   immediately integrate and use those as part of their products
-1. Developers of products and services can automatically provision supporting 
-   services such as task runners, package registries, deployment orchestrators and 
-   observability systems to build, verify, operate and observe their products
-1. Operators of third-party products and services can automatically provision 
-   spaces and supporting services to deploy and use those third-party products and 
-   services
-
-A more advanced platform also _composes_ these capabilities into experiences and 
-templates fit for key scenarios like web application development or data analysis. 
-For example, an application developer platform can include templates for provisioning 
-complete development environments and could serve the following use cases:
+At its more basic, a platform is a **product** developed by a 
+platform engineering team that is custom built and composed of supporting 
+reusable capabilities and services designed to reduce the cognitive load and burden 
+experienced by **internal** stakeholders of an enterprise organization 
+while still safely shipping software at speed [[4]]. 
+
+Note that while internal stakeholders typically refer to developers, they can 
+refer to DevOps, SREs, FinOps, Customer Success, and anyone who is responsible 
+for software delivery and cloud native infrastructure in an enterprise organization [[2]].
+
+The goal of any platform should be to drive 
+organizational efficiency and productivity by empowering internal stakeholders to 
+focus on their key responsibilities while giving them the tools and self-service 
+necessary to drive their day to day work. In order to discover those needs, a platform as product
+approach is needed which means starting off with building only what is necessary and then evolving 
+the platform over time with user feedback and usage metrics. For example, a "platform" may start off as a wiki page 
+with links to provision each required capability. Some [[3]] have referred to this principle as seeking
+the "thinnest viable platform" that meets an enterprise's requirements. 
+
+As the user needs of internal stakeholders evolve in an organization and platform owners get more used feedback
+and success metrics, platform engineering teams in an enterprise must take ownership of delivering the required capabilities. 
+This typically leads into the evolution of the thinnest viable platform into an **internal platform** in cloud-native computing 
```

Comment:
I don't think TVP _leads_ to internal platforms. Rather internal platforms start out as thinly as possible. TVP is really only applicable to an internal platform - an external platform by definition has to cater to many organizations' needs.

I've suggested TVP is related to "internal" but I'm still thinking :thinking: how to express that and work in "internal" here!

Reply:
Just want to see if we are on the same page with TVP as not just how something starts, but how it continues. I believe the industry already has a term for starting out small and that would be MVP (Minimum Viable Product). My understanding is that TVP evolves that to say, in the world of platforms, it is best to stay thin and continue to add value over as many widely available tools as possible.

So to expand further (since i won't be around to clarify 😅:
A thin platform may start minimal (only covering a few use cases) but grow past minimal by covering many use cases while staying thin by continuously looking for ways to outsource capabilities that were originally innovative and custom but have since become commodity in the industry.